EZP-31211 Document how to add a button for editing content in the Back Office (#947)
ezsystems/developer-documentation: docs/tutorials/extending_admin_ui/3_filtering_the_content_list.md

# Step 3a - Filtering the content list

> In this step you will enable the content list to be filtered by Content Types.  The following tutorial is based on the [Step 2 - Creating a content list](2_creating_a_content_list.md).

Reviewer: ```suggestion
In this step you will enable the content list to be filtered by Content Types.  The following tutorial is based on [Step 2 - Creating a content list](2_creating_a_content_list.md).
```

Reviewer: Also, not sure if this is completely clear, how about:
```suggestion
In this step you will enable the content list to be filtered by Content Types.  The following tutorial requires the completed [Step 2 - Creating a content list](2_creating_a_content_list.md).
```
